kairos-cli/src/lib.rs

```diff
@@ -1,4 +1,44 @@
+pub mod client;
 pub mod commands;
 pub mod common;
 pub mod error;
 pub mod utils;
+
+use crate::error::CliError;
+
+use clap::{Parser, Subcommand};
+use reqwest::Url;
+
+#[derive(Parser)]
+#[command(name = "Kairos Client", about = "CLI for interacting with Kairos")]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Command,
+    #[arg(long, value_name = "URL")]
+    pub kairos_server_address: Option<Url>,
```

Comment:
We always need server address, thus it should not be an `Option`.

**[Request]** Use the default value, for example:

```suggestion
    #[arg(long, value_name = "URL", default_value_t = get_default_api_address())]
    pub kairos_server_address: Url,
```

It would simplify `kairos_cli::run` - there is no longer need to destruct `Cli` structure.

Reply:
Using `default_value` now [987177d](https://github.com/cspr-rad/kairos/pull/91/commits/987177d0020214b76a8d3f346e516245a12f5e86)